app/Controllers/Account/JobController.php

```diff
@@ -53,4 +64,50 @@ public function show()
 
         return view('account/jobs/show', compact('job'));
     }
+
+    public function edit()
+    {
+        $categories = Category::all();
+        $cities = City::all();
+        $companies = Company::query()
+            ->whereBelongsTo(auth()->user())
+            ->get();
+
+        $job = Job::findOrFail(request()->query('id'));
```

Comment:
Này nó lấy toàn bộ luôn à, phải query theo account_id

Reply:
![image](https://user-images.githubusercontent.com/92205964/220959490-fa99a097-87fc-4b35-a662-b49a29c3a8d5.png)
như z hả

Reply:
where('account_id, ....) tương tự như whereBelongsTo(...